src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java

```diff
@@ -67,10 +67,14 @@ public AboutToStartOrSubmitResponse<CaseData, State> abutToSubmit(CaseDetails<Ca
     }
 
     public SubmittedCallbackResponse partiesContacted(CaseDetails<CaseData, State> details,
-                                                  CaseDetails<CaseData, State> beforeDetails) {
+                                                      CaseDetails<CaseData, State> beforeDetails) {
+
+
         return SubmittedCallbackResponse.builder()
-            .confirmationHeader(format("# Message sent. %n##  A notification has been sent via email to:Subject , Respondent."))
+            .confirmationHeader(format("# Message sent. %n##  A notification has been sent via email to:"
```

Comment:
Dont we need  Subject, Respondent in the confirmation message. Please ignore this message if its all expected.